src/aggregator/aggregator/aggregator_test.go

```diff
@@ -1167,7 +1168,7 @@ func testOptions(ctrl *gomock.Controller) Options {
 	infiniteBufferForPastTimedMetricFn := func(time.Duration) time.Duration {
 		return math.MaxInt64
 	}
-	return NewOptions().
+	return NewOptions(clock.NewOptions()).
```

Comment:
Nit: coud we use `newTestOptions()` here as well?

Reply:
https://github.com/m3db/m3/pull/3382/commits/f7533157ff0293ede5c77696b6f238dcdfeaa1e2